glmmTMB/tests/testthat/test-VarCorr.R

```diff
@@ -198,3 +198,16 @@ test_that("VarCorr omits resid when dispformula=~0", {
 test_that("vcov(.,full=TRUE) works for zero-disp models", {
     expect_equal(dim(vcov(fm0,full=TRUE)),c(4,4))
 })
+
+test_that("blockCode set correctly in VarCorr", {
+    set.seed(102)
+    n <- 25                                              ## Number of time points
+    x <- MASS::mvrnorm(mu = rep(0,n),
+                       Sigma = .7 ^ as.matrix(dist(1:n)) )    ## Simulate the process using the MASS package
+    y <- x + rnorm(n)                                   ## Add measurement noise
+    times <- factor(1:n, levels=1:n)
+    group <- factor(rep(1,n))
+    dat0 <- data.frame(y, times, group)
+    model <- glmmTMB(y ~ ar1(factor(times) + 0 | group), ziformula = ~ ar1(factor(times) + 0 | group), data=dat0, family = gaussian())
```

Comment:
This is giving a warning on my computer. 
```
Warning ('test-VarCorr.R:211:5'): blockCode set correctly in VarCorr
Model convergence problem; non-positive-definite Hessian matrix. See vignette('troubleshooting')
Backtrace:
    ▆
 1. └─glmmTMB::glmmTMB(...) at test-VarCorr.R:211:5
 2.   └─glmmTMB::fitTMB(TMBStruc) at glmmTMB/R/glmmTMB.R:1329:5
 3.     └─glmmTMB::finalizeTMB(TMBStruc, obj, fit, h, data.tmb.old) at glmmTMB/R/glmmTMB.R:1735:5

``` 
It works if I decrease the measurement noise on line 207 `y <- x + rnorm(n, sd = 0.1)`. Higher values of `sd` gave the same warning. Is this OK?

Reply:
I think that's fine (this is just a made-up example to test the printing machinery).